boards/nxp/imx95_evk/imx95_evk_mimx9596_m7.dts

```diff
@@ -148,3 +167,38 @@
 &edma2 {
 	status = "okay";
 };
+
+&dpu {
+	status = "okay";
+	width = <1080>;
+	height = <2340>;
+	hsw = <4>;
+	hfp = <12>;
+	hbp = <8>;
+	vsw = <4>;
+	vfp = <10>;
+	vbp = <10>;
+};
+
+&mipi_dsi {
+	status = "okay";
+	compatible = "nxp,mipi-dsi-dwc";
+	dpi-color-coding = "24-bit";
+	dpi-video-mode = "non-burst-sync-pulse";
+	dphy-ref-frequency = <24000000>;
+	data-rate-clock = <890666664>;
+	nxp,dc = <&dpu>;
+	width = <1080>;
+	height = <2340>;
+	rm692c9@0 {
+		status = "okay";
+		compatible = "raydium,rm692c9";
+		reg = <0x0>;
+		width = <1080>;
+		height = <2340>;
+		reset-gpios = <&gpio_exp0 8 GPIO_ACTIVE_HIGH>;
+		bl-gpios = <&gpio_exp0 4 GPIO_ACTIVE_HIGH>;
+		data-lanes = <4>;
+		pixel-format = <0>;
+	};
+};
```

Comment:
I would suggest referring to some shields we have for existing displays from raydium and add a similar shield
https://github.com/zephyrproject-rtos/zephyr/tree/main/boards/shields/rk055hdmipi4m
https://github.com/zephyrproject-rtos/zephyr/tree/main/boards/shields/g1120b0mipi